commands/waitForUrlToContain.js

```diff
@@ -0,0 +1,81 @@
+var util = require('util'),
+    events = require('events');
+
+var WaitForUrlToContain = function() {
+    events.EventEmitter.call(this);
+    this.startTimer = null;
+    this.cb = null;
+    this.ms = null;
+    this.selector = null;
+    this.protocol = require('nightwatch/lib/api/protocol.js')(this.client);
+};
+
+util.inherits(WaitForUrlToContain, events.EventEmitter);
+
+/**
+ * Waiting for url expected
+ * @param  {string} url [url expected]
+ * @param  {number} milliseconds [time for close assert]
```

Comment:
Since renaming these params would mean refactoring across several commands, let's just update these comments to be clear that `milliseconds` means overall command timeout, and `timeout` means a delay before we start polling the URL for what we expect it to contain.